test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -139,23 +143,28 @@ TEST_F(TxPipelineIntegrationTest, GetTransactionsTest) {
   ASSERT_EQ(given_tx, *PbTransactionFactory{}.deserialize(got_pb_tx));
 }
 
+constexpr auto kUser = "user@test";
+constexpr auto kAsset = "asset#domain";
+
 /**
  * @given unsigned empty GetAccount query
  * AND default-initialized IntegrationTestFramework
  * @when query is sent to the framework
  * @then query response is STATELESS_INVALID
  */
 TEST(PipelineIntegrationTest, SendQueryWithValidation) {
-  iroha::model::GetAccount query;
+  auto query = shared_model::proto::QueryBuilder()
+                   .createdTime(iroha::time::now())
+                   .creatorAccountId(kUser)
+                   .queryCounter(1)
+                   .getAccount(kUser)
+                   .build()
+                   .signAndAddSignature(
+                       shared_model::crypto::DefaultCryptoAlgorithmType::
+                           generateKeypair());
   integration_framework::IntegrationTestFramework()
       .setInitialState()
-      .sendQuery(query,
-                 [](const auto &res) {
-                   auto err_res =
-                       dynamic_cast<const iroha::model::ErrorResponse &>(res);
-                   ASSERT_EQ(iroha::model::ErrorResponse::STATELESS_INVALID,
-                             err_res.reason);
-                 })
+      .sendQuery(query)
```

Comment:
The test name is `SendQueryWithValidation`, but validation is missing.

Reply:
That's because of [IR-866](https://soramitsu.atlassian.net/projects/IR/issues/IR-866), I think test name can be changed

Reply:
Description should also be changed in this case, since "query response is STATELESS_INVALID" is not verified. Add TODO and IR-866 reference.

Reply:
Fixed in aeb44d3